README.md

```diff
@@ -163,6 +163,10 @@ Everyone using Jupyter notebooks should be doing this.
 
 > thanks a lot for your effort to create such a useful tool
 
+**Simon Brugman**, [Co-Founder of GraphKite](https://github.com/nbQA-dev/nbQA/pull/490#issue-529173596):
```

Comment:
@MarcoGorelli "Data scientist & pandas-profiling dev" would be more fitting here :)

Reply:
Sure, sorry about that, will about just now!